docs/build/wasm/swanky.md

```diff
@@ -287,15 +293,20 @@ Swanky Node is a Substrate based blockchain configured to enable `pallet-contrac
 Swanky Node is optimized for local development, while removing unnecessary components such as P2P. Additional features and pallets, such as to interact between (Contract <-> Runtime), will be added in the future.
 
 ### Compatible ink! version
+
 ink! `v4.0.0-beta` or lower is supported by pallet-contract polkadot-0.9.33 branch.
 
 ### Installation
+
 #### Download Binary
+
 The easiest method of installation is by downloading and executing a precompiled binary from the [Release Page](https://github.com/AstarNetwork/swanky-node/releases)
 
 #### Build Locally
+
 If you would like to build the source locally, you should first complete the [basic Rust setup instructions](https://github.com/AstarNetwork/swanky-node/blob/main/docs/rust-setup.md).
 Once Rust is installed and configured, you will be able to build the node with:
```

Comment:
is this link to github necessary? Maybe using docs link would make more sense, since in this way you need to be sure github docs are valid and up to date as well
`/docs/build/environment/ink_environment#rust-and-cargo`